testutils/source.go

```diff
@@ -795,22 +795,6 @@ func main() {
 	})
 	log.Fatal(http.ListenAndServe(":3000", nil))
 }`}, 1, gosec.NewConfig()}, {[]string{`
-package main
```

Comment:
Do you mean this is a duplicate of:

```Go
package main
import (
"os"
"io/ioutil"
"log"
)
func main() {
f := os.Getenv("tainted_file")
body, err := ioutil.ReadFile(f)
if err != nil {
 log.Printf("Error: %v\n", err)
}
log.Print(body)
```

I think the `ReadFile` has a string concatenation in the path: `"/tmp/" + f2`.

Reply:
Yes, you are right.

My mistake.